app/src/app/components/main/hooks/useGetPacketById.ts

```diff
@@ -2,10 +2,11 @@ import useSWR from "swr";
 import appConfig from "@config/appConfig";
 import { fetcher } from "@lib/fetch";
 import { PacketMetadata } from "@/types";
+import { packetIdRegex } from "@components/contents/admin/pins/utils/constants";
 
 export const useGetPacketById = (packetId: string | undefined) => {
   const { data, isLoading, error } = useSWR<PacketMetadata>(
-    packetId ? `${appConfig.apiUrl()}/packets/${packetId}` : null,
+    packetId && packetIdRegex.test(packetId) ? `${appConfig.apiUrl()}/packets/${packetId}` : null,
```

Comment:
I added this check here because otherwise every letter typed into the packet id input in the AddPinForm triggers a new request checking that there exists a packet with that id - but there never would be if the id isn't even well-formed, so we block those off at the pass.